services/app-api/utils/validation/completionSchemas.ts

```diff
@@ -17,6 +17,7 @@ export const error = {
   INVALID_DATE: "Response must be a valid date",
   INVALID_END_DATE: "End date can't be before start date",
   INVALID_NUMBER: "Response must be a valid number",
+  NON_POSITIVE_NUMBER: "Response must be a positive number",
```

Comment:
Looking good! Just a quick question about the phrasing, wondering if this is clear for users that 0 is not included. Would you mind asking Kim or Megan about that?

Reply:
Good point, thanks ill check.

Reply:
@karla-vm "Response must be a number greater than zero"  confirmed by Megan, Kim & DG